tests/test_block/test_create_delete_update_block_error.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import os
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.block.create_block_fixtures import (
+    create_block_query,
+    update_block,
+    delete_block,
+)
+
+
+sys.path.append(os.getcwd())
+
+
+class TestCreateBlockError(BaseTestCase):
+
+    def test_databse_connection_error(self):
```

Comment:
Please fix the typo in this function name

Reply:
Done @michael-basweti